vars/cosaParallelCmds.groovy

```diff
@@ -0,0 +1,27 @@
+// Run cosa commands in parallel
+// Available parameters:
+//    cosaDir:        string    -- cosa working directory
+//    isBuild:        boolean   -- run build commands using `cosa buildextend-`
+//                                 e.g pass to runCommands only:
+//                                 ['metal', 'metal4k', 'live', 'AWS']
```

Comment:
Do you expect to use this for commands which aren't `buildextend-...`? I think any other command doesn't parallelize as obviously.